main.py

```diff
@@ -20,7 +20,7 @@
 from subtest import scenario
 from subtest import utils
 
-DEFAULT_CONFIG_FILE = "./config.yml"
+DEFAULT_CONFIG_FILE = "./config_quick_debug.yml"
```

Comment:
I understood that it should remain ./config.yml, did I miss something ?

Reply:
I agree @arthurPignet , I suppose @jeromechambost you did that to facilitate debugging right? 😃